### PR TITLE
[october 15 2021] allowlisting a bunch of sites from /issues

### LINKED ISSUE #4922: trinity.jp
this is just a smartphone accessory brand

### LINKED ISSUE #5158: ALTUS.FINANCE - WRONGFULLY LISTED FOR ETH PHISHING
Hello,

We are the management team behind the website: https://altus.finance/

It has come to our attention that we have been placed on the MetaMask's phishing site list. This came to us as a surpise as we offer no crypto wallet custodian service nor any exchange facilities of any sort. Our business centers on providing our subscribers with valuable cryptocurrency content over Telegram.

We would highly apreciate if we could get this matter resolved swiftly as we hope to launch our service in the coming days.

Kind Regards

ALTUS

### LINKED ISSUE #5259: Blocking Issue: Metamade.io
Metamask is blocking our domain metamade.io

### LINKED ISSUE #5355: Altus.Finance
Hello,

We are the management team behind the website: https://altus.finance/

It has come to our attention that we have been placed on the MetaMask's phishing site list. This came to us as a surpise as we offer no crypto wallet custodian service nor any exchange facilities of any sort. Our business centers on providing our subscribers with valuable cryptocurrency content over Telegram.

We would highly apreciate if we could get this matter resolved swiftly as we hope to launch our service in the coming days.

Kind Regards

ALTUS

### LINKED ISSUE #5454: trinity.jp was blocked by mistake
trinity.jp is not Phishing site.

https://trinity.jp

Please remove from blocking list.

### LINKED ISSUE #5564: Ethereum Phishing Detection warning for opensfm.org
Searching in 
https://cryptoscamdb.org/scams
https://metamask.github.io/eth-phishing-detect/

returns no hits (empty list or "This domain is not blocked! No problem here.")

So it appears **opensfm.org** was blocked by mistake?

### LINKED ISSUE #5565: incorrectly flagged https://opentek.eu
a web based citizen science platform to collect local indicators of climate change impacts

### LINKED ISSUE #5566: Open.Ga.Gov
The website https://open.georgia.gov/description.html = https://open.ga.gov/description.html are NOT spam or phishing site. They just don't use an https protocol, perhaps?

### PR DESCRIPTION
**Pull request by gravelcycles**
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5583
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5634
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5633
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5631
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5643
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5629
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5601
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5626
Fixes https://github.com/MetaMask/eth-phishing-detect/issues/5625
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5622
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5598
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5596
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5575
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5621
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5620
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5615
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5619
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5618
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5617
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5606
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5605
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5454
fixes https://github.com/MetaMask/eth-phishing-detect/issues/4922
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5602
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5600
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5594
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5582
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5574
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5573
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5572
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5259
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5565
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5566
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5564
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5355
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5158
fixes https://github.com/MetaMask/eth-phishing-detect/issues/5571

### CODE REVIEW

**Comment by davidwgrossman** [comment]
> Can you link the issue numbers (ie: `Fixes #01` so it will autoclose issues on merge) pls

I did not know about the autoclosing feature! Yes, will do

File: src/config.json [hunk 1]
```diff
@@ -19,6 +19,32 @@
     "launchpad.ethereum.org"
   ],
   "whitelist": [
+    "opengear.com",
+    "opinsta.com",
+    "opensc.org",
+    "affinity.locus",
+    "opendev.org",
+    "opensee.io",
+    "openlca.org",
```
Comment (Gudahtt) [inline src/config.json:28]: This domain isn't mentioned in any issues :thinking:

Comment (radarzeo) [reply src/config.json:28]: I report 13 days ago 
https://github.com/MetaMask/eth-phishing-detect/issues/5574

Domain punctus.org
Hope you can whitelist this domain, thanks!

Comment (gravelcycles) [reply src/config.json:28]: @Gudahtt I believe it's mentioned in #5625

Comment (gravelcycles) [reply src/config.json:28]: @radarzeo please look at line # 41

Comment (Gudahtt) [reply src/config.json:28]: Sorry I'm not sure how I missed that!

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**